pkg/jobrunaggregator/jobrunhistoricaldataanalyzer/analyzer.go

```diff
@@ -54,9 +54,9 @@ func (o *JobRunHistoricalDataAnalyzerOptions) Run(ctx context.Context) error {
 
 	switch {
 	case o.newFile == "" && o.dataType == "alerts":
-		newHistoricalData = o.getAlertData(ctx)
+		newHistoricalData, err = o.getAlertData(ctx)
 		if newHistoricalData == nil {
-			return fmt.Errorf("Failed while attempting to read Alert Historical Data")
+			return fmt.Errorf("failed while attempting to read Alert Historical Data: %w", err)
```

Comment:
`errors.WithMessage(err, "failed while attempting to read Alert Historical Data")`  is the new way to do things

Reply:
noted.  and thanks!